content/en/docs/concepts/containers/images.md

```diff
@@ -47,6 +47,19 @@ to roll back to a working version.
 Instead, specify a meaningful tag such as `v1.42.0`.
 {{< /caution >}}
 
+### ImagePullBackOff
```

Comment:
```suggestion
### ImagePullBackOff

```

Reply:
Hey! Thanks for the PR, it seems to answer my original question.

I suggest to move the "ImagePullBackOff" section under "Updating images", instead of "Image names".

Reply:
Sure will do the changes.

Reply:
I have made the changes. Please review!